### PR TITLE
feat: [SRM-15097]: Enable metric-threshold for service health

### PR DESCRIPTION
**Pull request by deepesh-ui**
#### Screenshots

Screenshot 2023-07-10 at 12 26 23 PM

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/85-cv/pages/health-source/common/MetricThresholds/MetricThresholds.utils.ts [hunk 1]
```diff
@@ -104,7 +106,7 @@ export function getCustomMetricGroupNames(groupedCreatedMetrics: GroupedCreatedM
     return []
   }
 
-  const groupsWithCVEnabled = getGroupsWithCVEnabled(groupedCreatedMetrics)
+  const groupsWithCVEnabled = getGroupsWithCVandServiceHealthEnabled(groupedCreatedMetrics)
```
Comment (praneshg239) [inline src/modules/85-cv/pages/health-source/common/MetricThresholds/MetricThresholds.utils.ts:109]: please rename `groupsWithCVEnabled` to `groupsWithCVAndServiceHealthEnabled`

File: src/modules/85-cv/pages/health-source/common/MetricThresholds/MetricThresholds.utils.ts [hunk 2]
```diff
@@ -362,7 +364,7 @@ export function getMetricTypeItems(
 }
 
 function getMetricsHaveCVEnabled(selectedMetricDetails: GroupedMetric[]): GroupedMetric[] {
```
Comment (praneshg239) [inline src/modules/85-cv/pages/health-source/common/MetricThresholds/MetricThresholds.utils.ts:366]: shall we rename this to `getMetricsHaveCVAndServiceHealthEnabled`?

File: src/modules/85-cv/pages/health-source/common/MetricThresholds/MetricThresholds.utils.ts (hunk 1 above)
Comment (deepesh-ui) [reply src/modules/85-cv/pages/health-source/common/MetricThresholds/MetricThresholds.utils.ts]: done

File: src/modules/85-cv/pages/health-source/common/MetricThresholds/MetricThresholds.utils.ts (hunk 2 above)
Comment (deepesh-ui) [reply src/modules/85-cv/pages/health-source/common/MetricThresholds/MetricThresholds.utils.ts]: done

**Review by praneshg239 — APPROVED** [review]

**Review by karansaraswat19 — APPROVED** [review]

**Pull request merged**